site/docs/numbers/setup-reports/accountReports.mdx

```diff
@@ -0,0 +1,344 @@
+            </Values>
+            <Description>Report Parameter Description</Description>
+            <MultiSelectAllowed>true</MultiSelectAllowed>
+            <HelpInformation>Report Parameter Help Text</HelpInformation>
+         </Parameter>
+      </Parameters>
+   </Report>
+</ReportResponse>
+```
+
+</TabItem>
+</Tabs>
+
+## Request a Report {#request-account-report}
+
+To generate a specific report, you must make a <Highlight color="#079CEE">POST</Highlight> request to our [Create Report Instance API](../../../../../apis/numbers#tag/accounts/operation/CreateReportInstance) endpoint.
+Response will contain a location header used to query the status of the report.
+This can be done through tools like [Postman](https://github.com/Bandwidth/postman) or in the cURL.
+
+<Tabs
+groupId="generate_report"
+defaultValue="payload"
+values={[
+    { label: 'Payload', value: 'payload', },
+    { label: 'cURL', value: 'curl', }
+]
+}>
+
+<TabItem value="payload">
+
+<p><Highlight color="#079CEE">POST</Highlight><code>https://dashboard.bandwidth.com/api/accounts/{accountId}/reports/{reportId}/instancess</code></p>
+
+```xml
+<Instance>
+    <OutputFormat>pdf</OutputFormat>
+    <Parameters>
+        <Parameter>
+            <Name>Account</Name>
+            <Value>1</Value>
+        </Parameter>
+        <Parameter>
+            <Name>Sub-account</Name>
+            <Value>1</Value>
+        </Parameter>
+        <Parameter>
+            <Name>MultiValueEnum</Name>
+            <Value>value1</Value>
+            <Value>value2</Value>
+        </Parameter>
+    </Parameters>
+    <ExpiresAt>2020-11-25</ExpiresAt>
+</Instance>
+```
+</TabItem>
+
+<TabItem value="curl">
+
+```cURL
+curl -X POST 'https://dashboard.bandwidth.com/api/accounts/{accountId}/reports/{reportId}/instancess'
+-U '{userName}:{password}'
+-H 'Content-Type: application/xml'
+-d '<Instance>
+        <OutputFormat>pdf</OutputFormat>
+        <Parameters>
+            <Parameter>
+                <Name>Account</Name>
+                <Value>1</Value>
+            </Parameter>
+            <Parameter>
+                <Name>Sub-account</Name>
+                <Value>1</Value>
+            </Parameter>
+            <Parameter>
+                <Name>MultiValueEnum</Name>
+                <Value>value1</Value>
+                <Value>value2</Value>
+            </Parameter>
+        </Parameters>
+        <ExpiresAt>2020-11-25</ExpiresAt>
+    </Instance>'
+```
+
+</TabItem>
+</Tabs>
+
+> Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/xml; charset=utf-8
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/reports/{reportId}/instances/789
+```
+
+## Fetch Report Status
+
+Since the Account Report request generation is asynchronous, make a <Highlight color="#00bf8c">GET</Highlight> request to our [Retrieve Report Instance API](../../../../../apis/numbers#tag/Reports/operation/RetrieveReportInstance) endpoint to determine the status.
```

Comment:
```suggestion
Since the account report request generation is asynchronous, make a <Highlight color="#00bf8c">GET</Highlight> request to our [Retrieve Report Instance API](../../../../../apis/numbers#tag/Reports/operation/RetrieveReportInstance) endpoint to determine the status.
```